### PR TITLE
Main french translation

### PR DESCRIPTION
**Pull request by alyssaheinze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,58 @@ sed -i.bak 's|href="http|target="_blank" href="http|g' *.html
...
+rmarkdown::render("design/design.rmd",output_format='pdf_document')
+rmarkdown::render("effect-types/effect-types.Rmd",output_format='pdf_document')
+rmarkdown::render("external-validity/extval.Rmd",output_format='pdf_document')
+rmarkdown::render("hypothesistesting/hypothesistesting.Rmd",output_format='pdf_document')
+rmarkdown::render("measurement/measurement.Rmd",output_format='pdf_document')
+rmarkdown::render("mechanisms/10thingsaboutmechanisms.Rmd",output_format='pdf_document')
+rmarkdown::render("pap/pap.Rmd",output_format='pdf_document')
+rmarkdown::render("power/power.Rmd",output_format='pdf_document')
+rmarkdown::render("x-cause-y/x-cause-y.Rmd",output_format='pdf_document')
+rmarkdown::render("randomization/randomization.Rmd",output_format='pdf_document')
```
Comment (alyssaheinze) [inline README.md]: I could remove this

Comment (alyssaheinze) [reply README.md]: done

**Comment by jwbowers** [comment]
@alyssaheinze  Although I think that you can make your `main` branch be the French-translation, my instinct is for the guides stored on the Egap site to be labeled something like main-french-translation.  So, the idea would be to not merge into Egap's own main branch. Here, I think you are just making a pull request into your own repository. And then later your `main` will be `main-french-translation` in the overall Egap site. Is that your plan?

**Comment by alyssaheinze** [comment]
@jwbowers 
Thanks for your suggestion.
I close this PR.
I just created another PR on egap:main-french.
https://github.com/egap/methods-guides/pull/44

Can you please check if it is good for you?

**Pull request closed without merging**